local/o365/classes/feature/courserequest/main.php

```diff
@@ -124,18 +124,16 @@ public function get_unmatched_teams_by_user_oid(string $userobjectid) {
         }
 
         // Get Teams.
-        $subtypes = ['teamfromgroup', 'course', 'courseteam'];
-        [$subtypesql, $subtypeparams] = $DB->get_in_or_equal($subtypes);
         $teamsoidsql =
             "SELECT distinct objectid
                FROM {local_o365_objects}
-              WHERE type = ?
-                AND subtype $subtypesql";
-        $matchedgroupoids = $DB->get_fieldset_sql($teamsoidsql, array_merge(['group'], $subtypeparams));
+              WHERE type = 'group'
+                AND subtype IN ('teamfromgroup', 'course', 'courseteam')";
+        $matchedgroupoids = $DB->get_fieldset_sql($teamsoidsql);
 
         // Get SDS courses.
-        $sdsmatchedgroupoids = $DB->get_fieldset_select('local_o365_objects', 'objectid', 'type = ? AND subtype = ?',
-            ['sdssection', 'course']);
+        $sdsmatchedgroupoids = $DB->get_fieldset_select('local_o365_objects', 'objectid',
+            'type = "sdssection" AND subtype = "course"');
```

Comment:
These double-quotes aren't valid for PostgreSQL. There's many other instances in this commit too.
```
ERROR:  column "sdssection" does not exist
LINE 1: ...objectid FROM mdl_local_o365_objects WHERE type = "sdssectio...
                                                             ^
```

Reply:
@usqfowlerj Thank you so much for reporting the issue. We have made a new release containing the fix. Please check it out.